app/views/visualizations/new.html.erb

```diff
@@ -0,0 +1,82 @@
+<h1>Visualizations#new</h1>
+<p>Find me in app/views/visualizations/new.html.erb</p>
+
+<h1>Create New Visualization</h1>
+
+<div class="row">
+   <div class="col-md-6 col-md-offset-3">
```

Comment:
I'm not sure where this is happening, but on my computer when the page is rendered fullscreen the form gets squished too small and all the words are overflow rendered. Is this something we should worry about? I've attached two files; the first with correct form display and the second with text overflow.
<img width="675" alt="Screen Shot 2020-10-04 at 3 31 58 PM" src="https://user-images.githubusercontent.com/44452004/95025309-be9f5800-0656-11eb-992d-6f1209b9df94.png">
<img width="999" alt="Screen Shot 2020-10-04 at 3 32 07 PM" src="https://user-images.githubusercontent.com/44452004/95025313-c3fca280-0656-11eb-982c-b2febb0d280e.png">

Reply:
I was having some trouble with this too, but I thought I fixed it (at least for how it gets rendered on my screen). I don't think this will be too hard to fully fix down the line, so I don't think we have to worry about it. We can keep it for now and take care of it later, but it's definitely good to know that we need to do something down the line.

Reply:
Okay, that's about what I was expecting, and I agree with you that we can fix this down the line and that it's not a top priority issue.